cmd/traffic/cmd/manager/internal/cluster/info.go

```diff
@@ -213,6 +213,23 @@ func NewInfo(ctx context.Context) Info {
 		dlog.Warnf(ctx, "Unable to get manager pod ip; env var says %s", env.PodIP)
 	}
 
+	oi.DnsConfig = &rpc.DNSConfig{}
+	oi.DnsConfig.AlsoProxySubnets, err = env.GetAlsoProxySubnets()
+	if err != nil {
+		dlog.Warnf(ctx, "AlsoProxySubnets not parsed: %v", err)
+	}
+	for _, subnet := range oi.DnsConfig.AlsoProxySubnets {
+		dlog.Infof(ctx, "Storing AlsoProxySubnet: %+v", subnet)
```

Comment:
I suggest not saying "Storing" but just something like "Configured with AlsoProxySubnet" or similar, just to make it real easy for someone looking through the logs to feel confident they set it up right